### PR TITLE
Resilience Messaging - store all messages data in field at Nurse CT

### LINKED ISSUE #1212: All resilience messages shown before their due date
## Issue

All resilience messages are showing even though they are not due yet. 

According to @Diama1 , this is only happening in the pantheon environments and not on live 

# Task
- check and resolve

### PR DESCRIPTION
**Pull request by anvmn**
#1136

Also, fixes #1212  and changes from #1192

### Deployment instructions:

`drush updb`

### CODE REVIEW

**Comment by anvmn** [comment]
Messages data is stored at Nurse node:

Selection_511

File: client/src/elm/Pages/PinCode/View.elm [hunk 1]
```diff
@@ -346,6 +347,10 @@ viewLoggedInContent language currentTime features nurseId nurse ( healthCenterId
 
 resilienceReminderDialog : Language -> Time.Posix -> NominalDate -> NurseId -> Nurse -> Maybe (Html Msg)
 resilienceReminderDialog language currentTime currentDate nurseId nurse =
+    let
+        _ =
+            Debug.log "" nurse
+    in
```
Comment (mariano-dagostino) [inline client/src/elm/Pages/PinCode/View.elm]: Old code?

File: client/src/elm/Backend/ResilienceMessage/Utils.elm [hunk 2]
```diff
@@ -113,3 +117,205 @@ resilienceMessageOrderFromString value =
...
+        , ( ResilienceCategoryConnecting, 6 )
+        , ( ResilienceCategorySelfCare, 3 )
+        , ( ResilienceCategoryEndOfPeriod, 2 )
+        ]
+
+
+resolveDisplayDay : ResilienceCategory -> ResilienceMessageOrder -> Maybe Int
+resolveDisplayDay category order =
+    case category of
+        ResilienceCategoryIntroduction ->
```
Comment (mariano-dagostino) [inline client/src/elm/Backend/ResilienceMessage/Utils.elm:183]: :+1: I checked the same numbers against `hedley_resilience_create_messages`

**Review by mariano-dagostino — APPROVED** [review]

File: client/src/elm/Backend/ResilienceMessage/Utils.elm (hunk 2 above)
Comment (anvmn) [reply client/src/elm/Backend/ResilienceMessage/Utils.elm:183]: Right, logic moved from server to client,

File: client/src/elm/Pages/PinCode/View.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Pages/PinCode/View.elm]: Yep, will remove. Thanks.

**Review by mariano-dagostino — APPROVED** [review]

**Pull request merged**